hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/YarnCommands.md

```diff
@@ -122,8 +122,20 @@ Usage: `yarn logs -applicationId <application ID> [options] `
 | -applicationId \<application ID\> | Specifies an application id |
 | -appOwner \<AppOwner\> | AppOwner (assumed to be current user if not specified) |
 | -containerId \<ContainerId\> | ContainerId (must be specified if node address is specified) |
-| -help | Help |
 | -nodeAddress \<NodeAddress\> | NodeAddress in the format nodename:port (must be specified if container id is specified) |
+| -am \<AM Containers\> | Prints the AM Container logs, specify comma-separated value to get logs for related AM Container |
+| -client_max_retries \<Max Retries\> | Max number of retry for a client to get the container logs for the running applications |
+| -client_retry_interval_ms \<Retry Interval\> | Retry Interval (work with client_max_retries) |
+| -clusterId \<Cluster ID\> | ClusterId |
+| -list_nodes | Show the list of nodes that successfully aggregated logs (can only be used with finished applications) |
+| -log_files \<Log File Name\> | View a specific log type |
+| -log_files_pattern \<Log File Pattern\> | Get matched log files by using java regex |
+| -out \<Local Directory\> | Download logs to specified local folder |
+| -show_application_log_info | List all container IDs (combine this with --nodeAddress to get containerIds for all the containers on the specific NodeManager) |
+| -show_container_log_info | Show the container log metadata |
+| -size \<size\> | To list only a portion of the log files for a particular container |
+| -size_limit_mb \<Size Limit\> | Size of the total logs which could be fetched |
```

Comment:
Could you please rephrase these two sentences?
Both options limits the size of logs that is retrieved, but it is not mentioned that `-size` limits the number of bytes, and the `-size_limit_mb` limits the megabytes.